pkg/controllers/cspc-controller/sync.go

```diff
@@ -466,6 +466,7 @@ https://github.com/openebs/api/tree/master/design/cstor/v1
 ToDo: Offload this defaulting mechanism to mutating webhook server.
 */
 func defaultPoolConfig(cspi *cstor.CStorPoolInstance, cspc *cstor.CStorPoolCluster) {
+	defaultROThresholdLimit := 85
```

Comment:
we can move this const to the top of file with other const/var types.

Reply:
Done